lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1512,6 +1521,37 @@ OpFoldResult Aten__Getitem__DictStrOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Contains__IntListOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult Aten__Contains__IntListOp::fold(ArrayRef<Attribute> operands) {
+  auto arrayConstruct = getListConstructIfNotModified(l());
+  auto itemConstruct = item();
```

Comment:
nit: the convention in torch-mlir is to name variables corresponding to arguments of an op after the argument name, so this should be `Value item = item();`

Reply:
What do you mean by this? I already have a variable named item, should I rename it to something else?